packages/zoe/src/contracts/coveredCall-durable.js

```diff
@@ -0,0 +1,151 @@
+// @ts-check
+
+import { fit, M } from '@agoric/store';
+import { Far } from '@endo/marshal';
+import '../../exported.js';
+
+import {
+  defineDurableKind,
+  provideDurableMapStore,
+  provideKindHandle,
+} from '@agoric/vat-data';
+import { swapExact } from '../contractSupport/index.js';
+import { isAfterDeadlineExitRule } from '../typeGuards.js';
+
+const sellSeatExpiredMsg = `The covered call option is expired.`;
+
+/**
+ * A call option is the right (but not the obligation) to buy digital
```

Comment:
this comment is informative, but we already have it in `coveredCall.js`. What would be more helpful here is a `@see` tag pointing to other variants and explaining how and why they differ